src/mam4xx/convproc.hpp

```diff
@@ -608,14 +957,14 @@ void ma_precpevap(const Real dpdry_i, const Real evapc, const Real pr_flux,
 }
 
 //=========================================================================================
-template <typename SubView>
+template <typename Const_SubView, typename SubView>
```

Comment:
We might want to normalize the order of template parameters in these functions (e.g. so this function matches `update_tendency_final`). Then at least all the moving parts can be in similar orders. :-)

Reply:
Yes, I can make these templated functions more consistent in the declarations.